specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-08-02-preview/managedClusters.json

```diff
@@ -3943,6 +4024,10 @@
           {
             "value": "AvailabilitySet",
             "description": "Use of this is strongly discouraged."
+          },
+          {
+            "value": "Flex",
+            "description": "Create an Agent Pool that allows mixed-size vms, and would also scale in a non blocking mode."
```

Comment:
> and would also scale in a non blocking mode

i think only you and the devs of this feature understand what this means :D

Reply:
Removed.